tests/forms_tests/field_tests/test_jsonfield.py

```diff
@@ -97,6 +97,20 @@ class JSONForm(Form):
         form = JSONForm({'json_field': '["bar"]'}, initial={'json_field': ['foo']})
         self.assertIn('[&quot;foo&quot;]</textarea>', form.as_p())
 
+    def test_formfield_value_none(self):
+        class JSONForm(Form):
+            json_field = JSONField()
+
+        tests = [
+            {},
+            {'json_field': None},
+        ]
+        for data in tests:
+            with self.subTest(data=data):
+                form = JSONForm(data)
+                self.assertEqual(form["json_field"].value(), 'null')
```

Comment:
I don't see much value in this assertion we should check `as_p()` and `errors`, e.g.
```python
self.assertEqual(form.errors['json_field'], ['This field is required.'])
self.assertIn('null</textarea>', form.as_p())
```

Reply:
@felixxm I can remove it for sure, but on some level isn't the error due to `.value()` raising an error? Or is `.value()` too low level to test in general?

Reply:
We can leave an assertion for `value()`, but the two above assertions should also be added. It's not enough to check that `as_p()` is not `None`.

Reply:
Ok! I'll change the `self.assertIsNotNone(form.as_p())` to something that asserts the text.

Do you think we need to check the errors dict? The issue doesn't really have anything to do with validation from what I can gather, which is why I removed the `required=True` argument from the original PR, but I can re-add it if you want.

Reply:
> Ok! I'll change the `self.assertIsNotNone(form.as_p())` to something that asserts the text.

We don't need to assert the entire HTML, `self.assertIn('null</textarea>', form.as_p())` is enough.

> Do you think we need to check the errors dict? The issue doesn't really have anything to do with validation from what I can gather, which is why I removed the required=True argument from the original PR, but I can re-add it if you want.

Bound form is typically used due to invalid input, so IMO it makes sense to check `errors`.

Reply:
@felixxm Updated the asserts!